content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -212,7 +214,7 @@ At this point, the flow is now complete.
    policy, the down payments will **not** be able to be deducted when it comes time to invoice the
    customer.
 
-   This is because, due to the invoicing policy, the product(s) would have to be delivered *before*
+   This is because, due to the invoicing policy, the product would have to be delivered *before*
```

Comment:
```suggestion
   This is because, due to the invoicing policy, the product(s) would have to be delivered *before*
```
SUGGESTION: since there could be multiple products that need to be delivered, I believe using this parenetical is apt. However, this is totally your call 👍